### PR TITLE
CVSL-468: Specify cross-origin resource policy after Helmet upgrade

### PR DESCRIPTION
**Pull request by smcveigh941**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
     "@types/multer": "^1.4.7",
     "agentkeepalive": "^4.2.1",
     "applicationinsights": "^2.2.2",
-    "aws-sdk": "^2.1100.0",
+    "aws-sdk": "^2.1101.0",
```
Comment (smcveigh941) [inline package.json:98]: Minor version upgrade, was just published 2 days ago by AWS

File: package.json [hunk 2]
```diff
@@ -155,6 +154,7 @@
     "axe-core": "^4.4.1",
     "better-npm-audit": "^3.7.3",
     "cheerio": "^1.0.0-rc.10",
+    "concurrently": "^6.5.1",
```
Comment (smcveigh941) [inline package.json:157]: Was removed in error

File: package.json [hunk 3]
```diff
@@ -134,7 +134,6 @@
     "@types/compression": "^1.7.2",
     "@types/connect-flash": "0.0.37",
     "@types/connect-redis": "^0.0.18",
-    "@types/cookie-session": "^2.0.44",
```
Comment (smcveigh941) [inline package.json:137]: `cookie-session` was removed previously, but the associated types were not. `cookie-session` is not used, `express-session` is used instead

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**